kong/plugins/zipkin/opentracing.lua

```diff
@@ -172,11 +172,11 @@ function OpenTracingHandler:log(conf)
 			local span = proxy_span:start_child_span("kong.balancer", try.balancer_start / 1000)
 			span:set_tag(ip_tag(try.ip), try.ip)
 			span:set_tag("peer.port", try.port)
-			span:set_tag("kong.balancer.try", i)
+			span:set_tag("kong.balancer.try", string.format("%d", i))
 			if i < balancer_address.try_count then
 				span:set_tag("error", true)
 				span:set_tag("kong.balancer.state", try.state)
-				span:set_tag("kong.balancer.code", try.code)
+				span:set_tag("kong.balancer.code", string.format("%d", try.code))
```

Comment:
I don't see any reason to set these tag values as strings (it seems more natural to keep them as numbers)